### ISSUE TITLE
Manager slightly destroyed

### ISSUE THREAD

**Issue by inreti-sb** [bertoost/MODX-Polls#53]
Manager slightly destroyed (**light grey bar on top inserted**, some CSS statements like **padding of menu-item etc. not applying**) on **first request after flushing cache**.

Screenshot:
![Screenshot_2021-03-18 Dashboard polls bug](https://user-images.githubusercontent.com/4455364/111624698-4aafd100-87ec-11eb-9064-34980f5227f9.png)

On reload (from cache) everything is fine again.

In Developer Tools the **title- and meta-tags** and **all external CSS- and JS-links** seem to be **moved from head- to body-section**!

Even though this extra seems to be very old and unmaintained.. does anyone has an approach?

MODX Revolution 2.8.1-pl
polls 1.3.2-pl
PHP 7.4.16
Ubuntu 18.04.5 LTS
Apache 2

**Comment by Jako** [comment]
Since Raffy had the same solution for the problem, I looked through the code and this line can be the cause of the issue:

https://github.com/himurovich/MODX-Polls/blob/master/core/components/polls/controllers/mgr/polls_header.php#L14

There is no cleaning on the value …

It has to be removed there. After that `MODx.request` has to be used in the JS code of the extra replacing `Polls.request`.

**Comment by matdave** [comment]
@Jako can you test the PR I submitted. I was unable to "break" my manager, so can't recreate.

**Comment by Jako** [comment]
I don’t think that the Polls.request value is used somewhere. But I never had that extra installed on my own pages and just saw the result on another site. It was visible from time to time and seemed not reproducible. Maybe @inreti-sb can test it.

**Comment by inreti-sb** [comment]
Thanks @Jako  and @matdave! Unfortunately this PR does not fix it.
Does someone needs manager/server access to debug and test it?

**Comment by Jako** [comment]
Do you have a way to reproduce the issue?

**Comment by inreti-sb** [comment]
On three of my sites I tested so far (on two different v-servers).. install the extra "polls".. refresh page.. manager is slightly destroyed.

**Comment by Jako** [comment]
Installing can be done only once. Is it somehow else reproducible after? You told about the first request after clearing the cache. I just have to create a test situation, wether the issue is gone.

**Comment by inreti-sb** [comment]
I can reproduce this everytime I clear the cache (or reinstall the extra).. afterwards the manager is destroyed, but only on the first request.
On every further (cached) request and after uninstalling or deactivating the extra, the appearance is fine again.
I will try to reproduce this on a fresh instance without further extras installed.

**Comment by inreti-sb** [comment]
Okay.. I just reproduced this on a fresh 2.8.1 (advanced) installation with no additional extras. Hope that helps.

**Comment by Jako** [comment]
Can you check this patch?

**Comment by inreti-sb** [comment]
Jiiiha, this patch seem to work for me. Thanks a lot, @Jako!

**Comment by Jako** [comment]
@himurovich Can you merge the fix and release a new version?

**Comment by ant-tar** [comment]
Sure I will do @Jako Thanks a lot!
